AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1542,13 +1542,13 @@ petri.com#?#.content_widget:has(> div.widget_content > div.hs-form-insert)
 fort-russ.com#?#.single-container > div.better-studio-shortcode:has(> div.bs-push-noti-inner)
 gatevidyalay.com#?#aside > section.widget-sidebar:has(> div.textwidget > section > form[id^="subscribe-"])
 it-actual.ru##.widget-area > aside.widget > div[class] form[method="post"]
-thebell.io#?#.sidebar-item:has(> .connect)
-thebell.io#?#.sidebar-item:has(> div[id*="ad-sidebar-"])
-thebell.io#?#.sidebar-item:has(> div:contains(weekly newsletter))
+thebell.io##.sidebar
```

Comment:
too wide rule.

Reply:
On article page, sidebar contains only annoyances, we should remove it completely.

On front page, it also contain some annoyances and also some can-be-useful info. I prefer variant without sidebar completely, but we can probably hide it only on article page if this is important.

![thebell_without](https://user-images.githubusercontent.com/1848785/77298559-04ac5400-6ce3-11ea-80d0-962f6793ad61.png)
![thebell_with](https://user-images.githubusercontent.com/1848785/77298563-06761780-6ce3-11ea-9215-4714b9dd2cf3.png)

Reply:
ok